samples/RefreshFlow/AngularApp/protractor.conf.js

```diff
@@ -0,0 +1,32 @@
+// Protractor configuration file, see link for more information
```

Comment:
Are there tests for this sample project? If not, delete this test configuration file.